### PR TITLE
aligne Team 4171 with scaleUp

### PR DESCRIPTION
**Pull request by angelabriel**

### CODE REVIEW

**Review by fmherschel — COMMENTED** [review]
At least I have some questions inline (still ongoing)

File: SAPHana/ra/SAPHanaController [hunk 1]
```diff
@@ -96,15 +93,10 @@ function super_ocf_log() {
             none )
                 skip=1
                 ;;
-            * ) mtype=${message%% *}
+            * ) mtype=${message:0:4}
```
[the remark this answers is not in the document]
Comment (angelabriel) [reply SAPHana/ra/SAPHanaController:96]: As for now we only have filter with 3 characters, so I assume that was the reason Peter choose this approach.
But no problem, we can go back to a sequence like:

mtype=${message%% *}
mtype=${mtype%:}
[[ ${shf} == *${mtype}* ]] && skip=0 || skip=1

[the remark this answers is not in the document]
Comment (fmherschel) [reply SAPHana/ra/SAPHanaController:96]: FLOW is btw a 4 characters filter. It was never limitted to 3 chars.

File: SAPHana/ra/SAPHanaTopology [hunk 2]
```diff
@@ -76,15 +73,10 @@ function super_ocf_log() {
             none )
                 skip=1
                 ;;
-            * ) mtype=${message%% *}
+            * ) mtype=${message:0:4}
```
Comment (fmherschel) [inline SAPHana/ra/SAPHanaTopology:76]: Same as for SAPHanaConroller - limitation to 4 chars is not intended.

**Review by fmherschel — APPROVED** [review]
I approve the PR to unlock it. We should discuss if I re-allow log-purposes longer then 4 chars again.

**Pull request merged**